Dockerfile.assisted-service-build

```diff
@@ -11,7 +11,22 @@ ENV TOOLS=/tools/
 ENV PATH="$VIRTUAL_ENV/bin:$GOROOT/bin:$GOPATH/bin:$TOOLS:$PATH"
 
 COPY --from=quay.io/app-sre/golangci-lint:v1.46.0 /usr/bin/golangci-lint /usr/bin/golangci-lint
-COPY --from=quay.io/goswagger/swagger:v0.28.0 /usr/bin/swagger /usr/bin/goswagger
```

Comment:
you need to update here as well: https://github.com/openshift/assisted-service/blob/57ed23bb318c29daf89bde6db3e77257723574bc/ci-images/Dockerfile.code-generation#L6

Reply:
Thanks @osherdp, done.